### PR TITLE
Improve preferences activity style and add a large screen layout

### PR DESCRIPTION
**Pull request by BrayanDSO**
## Fixes
Part of #13878 

## Approach
In the commits

## How Has This Been Tested?

33 emulator:

[phone.webm](https://github.com/ankidroid/Anki-Android/assets/69634269/8f5dc43a-3340-4913-b0e4-463e35f16649)

[tablet.webm](https://github.com/ankidroid/Anki-Android/assets/69634269/13d71683-99f6-4330-88f8-120daa0b865d)

## Learning (optional, can help others)

You can't set a toolbar title directly if there is a CollpasingToolbarLayout https://stackoverflow.com/questions/26486730/in-android-app-toolbar-settitle-method-has-no-effect-application-name-is-shown

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by BrayanDSO** [comment]
Pending things that you may note:
1. The statusBar color is still light blue, which is something I want to fix in the future together with other things
2. I haven't removed the `screenSize` configChange from the manifest because this doesn't deal with screen resizes at the moment. If I assessed the work correctly, I'd need to create a custom backstack and fragment managers so I can propertly restore/replace the fragments when the screen is resized (i.e. too much work for a too small reward)

**Comment by criticalAY** [comment]
image
We should have this UI style (App showing along side the back icon) the current app bar is odd? The screen is scrolled leading to the actual way of doing it but initially the app bar and the title are not aligned

**Comment by BrayanDSO** [comment]
New Material 3 thing: https://m3.material.io/components/top-app-bar/specs. Have seen this in other apps, specially in iOS ones (notes, photos, app store, google agenda, PDF expert) and that's what I think that Material is perhaps trying to copy

**Comment by criticalAY** [comment]
Umm there are 4 options available depending upon the use case
image
wouldn't this cause an issue on the small screen, taking a little more space? 
I am fine with both 2 & 3 though

**Comment by BrayanDSO** [comment]
> wouldn't this cause an issue on the small screen, taking a little more space?

I don't think so, since it is compressible. Also, phone screen size average only gets bigger

**Comment by BrayanDSO** [comment]
For reference, I intend to make the title bold later, like other apps do (especially iOS ones), but I'd like to do this kind of change altogether (i.e. change font-weight in different places of the app)

e.g.
Pasted image 20230725194714

e.g. 2
image

**Comment by lukstbit** [comment]
Looks pretty good from what I saw but I really don't like the toolbar and empty space, seems wasteful. The main issue is that we don't have toolbar menu options to kind of "break" the space so its pretty continuous, this becomes more obvious in phone landscape or with drawing based languages(japanese or chinese) which use even smaller spaces for the Settings title. Also there will be phones were that extra space will force the user to scroll and we should avoid this.


My option for the toolbar is number 2 in the image above, but feel free to wait for other opinions as well.

**Comment by BrayanDSO** [comment]
> this becomes more obvious in phone landscape

I don't think that any of the app screens is good in phones landscape mode. Tablets/Chromebooks could benefit of it, but spending time adequating the app for phones landscape mode shouldn't be a goal.

> Also there will be phones were that extra space will force the user to scroll and we should avoid this.

We can, or better **should**, use low-dp layouts for those kind of cases. 

---
Also, I think that 3 is fine too. And would avoid 2 since it is too generic/bland IMHO

**Comment by criticalAY** [comment]
I personally like 2 but am fine with 3 also as M3 allows us to use that in small screens also

**Comment by BrayanDSO** [comment]
Seems pretty handy. I'll take a look at it. Thanks!

**Comment by BrayanDSO** [comment]
> Seems pretty handy. I'll take a look at it. Thanks!

Won't do. Doesn't fit exactly the layout, since in the tablets' layout the app bar is only above the lateral fragment, not above the main one. If I find an easy solution, I'll fix it eventually, but for now I don't think that it is worth the trouble

**Review by mikehardy — APPROVED** [review]
I think it looks great - with respect, I have read the qualitative disagreements, and they have merit but the defenses from Brayan were also pretty solid. It seemed like a good discussion, with a productive experiment in response even, despite abandoning it.

Given that the ideas appear to have been explored, the existing implementation looks pretty good in my personal opinion, others seem to accept it as at least acceptable, and Brayan - who is actually doing the work - likes it, I'm going to say we should maintain velocity here and get this in

File: AnkiDroid/src/main/res/layout-sw600dp/preferences.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (david-allison) [inline AnkiDroid/src/main/res/layout-sw600dp/preferences.xml:1]: no license

**Review by david-allison — CHANGES_REQUESTED** [review]
Consider this an 'approve' once license issues are discussed.

File: AnkiDroid/src/main/res/layout/preferences.xml [hunk 2]
```diff
@@ -1,40 +1,44 @@
-<?xml version="1.0" encoding="utf-8"?><!--
-  ~  Copyright (c) 2021 David Allison <davidallisongithub@gmail.com>
```
Comment (david-allison) [inline AnkiDroid/src/main/res/layout/preferences.xml:2]: License was removed - I don't mind removing my name (IDE default)

But we should stick with the GPL guidance

**Comment by BrayanDSO** [comment]
Licenses added

**Comment by mikehardy** [comment]
From David:

> Consider this an 'approve' once license issues are discussed.

From Brayan: 

>  Licenses added

👆 verified

**Pull request merged**